qiskit/transpiler/passes/basis/basis_translator.py

```diff
@@ -363,7 +363,7 @@ def _extract_basis_target(
         return source_basis, qargs_local_source_basis
 
 
-class StopIfBasisRewritable(Exception):
+class StopIfBasisRewritableError(Exception):
```

Comment:
This is explicitly not an error.  It's the success case lol.

Reply:
Reverted in any case.